tests/explorer.tests/ComponentTests.cs

```diff
@@ -40,6 +40,26 @@ static void TestResult(MinMaxRefiner.Result result)
             }
         }
 
+        [Fact]
+        public async Task TestMinMaxRefinerComponentWithNegativeValues()
+        {
+            using var scope = await testFixture.CreateTestScope("taxi", "jan08", "pickup_longitude", this);
+
+            // Construct MinMaxRefiner explicitly in order to inject a null result from MinMaxFromHistogramComponent
+            var histogramMinMaxProvider = new StaticResultProvider<MinMaxFromHistogramComponent.Result>(null!);
+            var refiner = new MinMaxRefiner(histogramMinMaxProvider) { Context = scope.Context };
+
+            TestResult(await refiner.ResultAsync);
+
+            static void TestResult(MinMaxRefiner.Result result)
+            {
+                const decimal aircloakMin = -106M;
+                const decimal aircloakMax = -6M;
```

Comment:
Could you get the min and max using a query instead of hard-coding these values here?

Reply:
You're right, this is much better. 

Done here 15c28bc.